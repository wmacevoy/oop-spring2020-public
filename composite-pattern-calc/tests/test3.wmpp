(((p+z)+s)+(((((g+(z*5))*((316+60)+7))+(571*((l*n)*0)))*2)*(((947*f)+(380+((2+k)*(3+p))))+(y*p))))
((x+((e*(((8*f)+0)+((4*m)*(4+u))))+7))*((163+((z+((4+7)+(9*0)))*(((1+798)*d)*(i+(d+3)))))*(y+(7+((k*1)+704)))))
((0*((149+(((k*0)*251)*((923*j)*n)))+(2*(k+((a+93)*(i+2))))))*(((v+(((6+c)+u)*((i+i)*(f*z))))+(5+(((4+119)+(s+i))+((2+y)+a))))*(((((j*816)*(7+7))*1)*(((9*t)+(y+a))*(s*(t*f))))*(3*6))))
(2*((((((s*7)*(6+9))*((4+y)*(u*961)))*6)+63)+(((((5*v)*(1+i))*((5+f)+(n*3)))+(((i+e)+(5+5))+(p+(403+0))))+((495+((i*8)+(5*l)))*2))))
((375*(308+((((d+e)+(w+2))*((m+r)*(a+x)))*((l+(9+7))*(m*(y+6))))))*((((((31+4)*(p+l))*609)*(((f+0)+k)*((0*7)+g)))+(j+(((1*h)*(v+32))+((h+s)*(4*m)))))*(((((2*m)*(h*0))+m)+(983*((p+3)*(2+z))))*((((r*2)+(p+3))+((7*i)*k))*((u*(704*d))+e)))))
(g*((((((7+w)*(5*v))*((n*h)*(459+c)))+y)+((d*((1+j)*(b*424)))*(((v*8)*(y+3))*((f*b)+(7+9)))))+(((((8*a)*(z*x))+((m+5)*i))+(((6+o)*7)+((y*508)+(104*h))))*((((m*4)*948)*((y+594)*(t+1)))*0))))
