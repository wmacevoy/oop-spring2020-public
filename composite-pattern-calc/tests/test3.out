15350.835
2136333.8
0.0
3293.6946
5.6746728E7
0.0015795724
